src/swap/components/Swap.tsx

```diff
@@ -140,6 +145,8 @@ export function Swap({ address, children, title = 'Swap' }: SwapReact) {
         return;
       }
       try {
+        /* when toAmount changes we fetch quote for fromAmount
```

Comment:
Small nit for future comments.

Use this kind of comments
```
//
```
inside function

and this kind
```
/**
 *
 */
```
at the begining of a function.